thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/onboard/tasks/FunctionCreationOnboardingTask.java

```diff
@@ -217,6 +218,8 @@ public void run() {
       anomalyFunction.setBucketSize(dataGranularity.getSize());
       anomalyFunction.setBucketUnit(dataGranularity.getUnit());
       anomalyFunction.setIsActive(configuration.getBoolean(IS_ACTIVE, DEFAULT_IS_ACTIVE));
+      anomalyFunction.setRequiresCompletenessCheck(configuration.getBoolean(REQUIRE_DATA_COMPLETENESS,
+          defaultFunctionSpec.isRequiresCompletenessCheck()));
 
       anoomalyFunctionDAO.update(anomalyFunction);
```

Comment:
Is this a typo?

Reply:
Nice catch. As this has been merged, will fix that in the nex pr.